sbi/utils/torchutils.py

```diff
@@ -247,7 +247,7 @@ def ensure_x_batched(x: Tensor) -> Tensor:
     return x
 
 
-def atleast_2d(*arys: Union[np.array, Tensor]) -> Union[Tensor, List[Tensor]]:
+def atleast_2d_many(*arys: Union[np.array, Tensor]) -> Union[Tensor, List[Tensor]]:
     """Return tensors with at least dimension 2.
```

Comment:
Might be nice to add that it calls `atleast_2d` in a for loop in the docstring.

Reply:
Agreed.